src/wpt/fs-test.ts

```diff
@@ -144,84 +145,73 @@ export default {
     {
-      comment: '...',
-      // TODO(node-fs): Inspect these tests
+      comment: 'We currently do not implement lock modes',
       disabledTests: true,
     },
   'FileSystemObserver.https.tentative.any.js': {
     comment: 'FileSystemObserver is not implemented in workers',
-    // TODO(node-fs): Implement FileSystemObserver?
     disabledTests: true,
   },
   'FileSystemObserver-sync-access-handle.https.tentative.worker.js': {
-    comment: '...',
-    // TODO(node-fs): Inspect these tests
+    comment:
+      'We currently do not implement sync access handles or FileSystemObserver',
     disabledTests: true,
   },
   'FileSystemObserver-unsupported-global.https.tentative.any.js': {
-    comment: '...',
+    comment: 'We do not implement FileSystemObserver',
+    disabledTests: true,
   },
   'FileSystemObserver-writable-file-stream.https.tentative.any.js': {
-    comment: '...',
-    expectedFailures: [
-      // TODO(node-fs): Fix these tests
-      'Closing a FileSystemWritableFileStream that\'s modified the file produces a "modified" event',
-      "All FileSystemWritableFileStream methods that aren't closed don't produce events",
-    ],
+    comment: 'We do not implement FileSystemObserver',
+    disabledTests: true,
   },
   'FileSystemSyncAccessHandle-close.https.worker.js': {
-    comment: '...',
-    // TODO(node-fs): Inspect these tests
+    comment: 'We currently do not implement sync access handles',
     disabledTests: true,
   },
   'FileSystemSyncAccessHandle-flush.https.worker.js': {
-    comment: '...',
-    // TODO(node-fs): Inspect these tests
+    comment: 'We currently do not implement sync access handles',
     disabledTests: true,
   },
   'FileSystemSyncAccessHandle-getSize.https.worker.js': {
-    comment: '...',
-    // TODO(node-fs): Inspect these tests
+    comment: 'We currently do not implement sync access handles',
     disabledTests: true,
   },
   'FileSystemSyncAccessHandle-read-write.https.worker.js': {
-    comment: '...',
-    // TODO(node-fs): Inspect these tests
+    comment: 'We currently do not implement sync access handles',
     disabledTests: true,
   },
   'FileSystemSyncAccessHandle-truncate.https.worker.js': {
-    comment: '...',
-    // TODO(node-fs): Inspect these tests
+    comment: 'We currently do not implement sync access handles',
     disabledTests: true,
   },
   'FileSystemWritableFileStream.https.any.js': {
-    comment: '...',
+    comment:
+      'There is currently a bug in the web platform tests that causes this to fail',
     expectedFailures: [
-      'createWritable() fails when parent directory is removed',
       'createWritable() can be called on two handles representing the same file',
     ],
   },
   'FileSystemWritableFileStream-piped.https.any.js': {
-    comment: '...',
-    // TODO(node-fs): Inspect these tests
+    comment: 'The test requires streams/resources/recording-streams.js',
```

Comment:
@npaun ... these tests need to be able to reach out to the streams wpts to get this file... have you encountered this in any other tests yet?